### DISCUSSION TITLE
Capture user input, including formatting and store in a variable for further processing

### DISCUSSION THREAD

**Discussion by jimmymcmahoncoding** [eternnoir/pyTelegramBotAPI#1627] [Q&A]
Good evening.

I hope that you are well.

I have a scenario where after running a command of /testcopymessage (or similar) I would like the user to be prompted to type or paste in a message.  This message will contain some formatted text (for example, some parts will be underlined, some parts will be in bold font etc) amongst standard plaintext.

Is it possible to capture the users message including the formatting that is applied and then send or forward that same formatted message to another chat_id?

I have seen the echo bot example, however that only appears to send an unformatted version of an inputted message (i.e. if I apply bold formatting to a message I want to send, the result is the correct message however the text is not bold formatted).

Many thanks in advance
James

**Comment by BasilNozdrin** [comment]
Try using markdown parse_mode when initializing bot. I'm not an expert, but I believe this should help as telegram is using mardown formatting.
